hera_qm/tests/test_xrfi.py

```diff
@@ -725,9 +726,10 @@ def test_xrfi_h1c_run_input_calfits(self):
         # input calfits
         uvd = UVData()
         uvd.read_miriad(test_d_file)
-        ext = '.flag'
-        outtest1 = os.path.join(xrfi_path, os.path.basename(test_c_file)) + '.x' + ext
-        outtest2 = os.path.join(xrfi_path, os.path.basename(test_c_file)) + '.g' + ext
+        ext = 'flag'
+        cbasename = os.path.basename(utils.strip_extension(test_c_file))
+        outtest1 = os.path.join(xrfi_path, cbasename) + '.x.' + ext
```

Comment:
there are a lot of lines you call something like: ` '.'.join([os.path.join(xrfi_path, basename), ext])` but this line and the next are different. I mean they both fulfill the same purpose and both work, just feels weird that it switched to a different coding style. Which is understandable since a lot of people have touched this code at some point.